.travis.yml

```diff
@@ -5,6 +5,7 @@ dist: trusty
 sudo: required
 
 env:
+  matrix:
```

Comment:
Yes, because I've added the global key. It's required to take differentiate between those two cases.